app/controllers/documents_controller.rb

```diff
@@ -108,4 +108,9 @@ def add_attachment
     end
     redirect_to :action => 'show', :id => @document
   end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.